CONTRIBUTING.md

```diff
@@ -4,26 +4,85 @@ Vald is an open source project.
 
 We appreciate your help!
 
-## Before filing an issue
+## Table of Contents
 
-If you are unsure whether you have found a bug, please consider asking in the [Vald Slack](https://join.slack.com/t/vald-community/shared_invite/zt-db2ky9o4-R_9p2sVp8xRwztVa8gfnPA) first. If the behavior you are seeing is confirmed as a bug or issue, it can easily be re-raised in the issue tracker.
+- [Contributing Issue](#Contributing-Issue)
+  - [Bug Report](#Bug-Report)
+  - [Proposal](#Proposal)
+  - [Feature Request](#Feature-Request)
+  - [Security Issue Report](#Security-Issue-Report)
+- [Contributing Source Code](#Contributing-Source-Code)
+  - [Before contributing source code](#Before-contributing-source-code)
+  - [How to contributing source code](#How-to-contributing-source-code)
+  - [Branch naming convention](#Branch-naming-convention)
 
-## Filing issues
+## Contributing Issue
 
-Sensitive security-related issues should be reported to the security channel in the [Vald Slack](https://join.slack.com/t/vald-community/shared_invite/zt-db2ky9o4-R_9p2sVp8xRwztVa8gfnPA) .
+We use [Github Issues](https://github.com/vdaas/vald/issues) to track issues within this repository.
+If you can determine the problem you are facing is a bug or issue, you can easily submit in the issues.
 
-Otherwise, when filing an issue, make sure to answer these five questions:
+If you are unsure whether you have found a bug or security-related issues, please consider asking in the [Vald Slack](https://join.slack.com/t/vald-community/shared_invite/zt-db2ky9o4-R_9p2sVp8xRwztVa8gfnPA) first. <br>
```

Comment:
```suggestion
If you are unsure whether you have found a bug or security-related issues, please consider asking in the [Vald Slack](https://join.slack.com/t/vald-community/shared_invite/zt-db2ky9o4-R_9p2sVp8xRwztVa8gfnPA) first. 
If the behavior you are seeing is confirmed as a bug or issue, it can easily be re-raised in the issue tracker.

```